build.sbt

```diff
@@ -34,33 +34,51 @@ lazy val docs = project
 val catsV = "2.3.1"
 
 val kindProjectorV = "0.11.2"
-val betterMonadicForV = "0.3.1"
+
+val Scala212 = "2.12.12"
+val Scala213 = "2.13.4"
+val Scala3 = Seq("3.0.0-M2", "3.0.0-M3")
+
+ThisBuild / crossScalaVersions := Seq(Scala212, Scala213) ++ Scala3
+ThisBuild / scalaVersion := Scala213
+ThisBuild / organization := "org.typelevel"
```

Comment:
I think this is correct, but please let me know if I should revert it to `chrisdavenport`

Reply:
This is correct, but we need the files to move to that package structure as well.